### DISCUSSION TITLE
LLOneBot下B站相关

### DISCUSSION THREAD

**Discussion by Annabel0530** [Tsuk1ko/CQ-picfinder-robot#458] [问题求助] [answered]
![image](https://github.com/Tsuk1ko/cq-picsearcher-bot/assets/40524485/a3995473-4a6a-490d-ac72-e24959fc582c)
动态不推送了，收到动态链接也回复错误，日志没有写入东西

**Comment by Tsuk1ko** [comment, marked as answer]
配个cookie，b站现在鉴权变严了

推送也建议用信息流推送

https://github.com/Tsuk1ko/cq-picsearcher-bot/wiki/%E9%99%84%E5%8A%A0%E5%8A%9F%E8%83%BD#%E4%BF%A1%E6%81%AF%E6%B5%81%E6%8E%A8%E9%80%81

**Reply by Annabel0530** [reply]
好，正常了目前
